### DISCUSSION TITLE
Make tar_visnetwork() widget bigger

### DISCUSSION THREAD

**Discussion by pvtodorov** [ropensci/targets#962] [Help]
Hi, I am working in a jupyter notebook using IRKernel. I normally use `options(repr.plot.width=15, repr.plot.height=15)` to set the rendered size of ggplot2 outputs but this does not work for the HTML widget output by `tar_visnetwork()` and I can't seem to figure out the right params to pass to change its size. Is there a way to do it?

**Comment by wlandau** [comment]
You could use `visOptions()` in [`visNetwork`](http://datastorm-open.github.io/visNetwork) directly:

```r
library(targets)
library(visNetwork)
tar_visnetwork() |>
  visOptions(width = "800px", height = "800px")
```

**Comment by mattwarkentin** [comment]
What timing. I was literally wondering about sizing the network, but @pvtodorov beat me to it.

I am wondering if there is a way to make the network flip to "horizontal"? I have a very "tall" pipeline at the moment, and it might be more aesthetic and easier to view the pipeline in its entirety if the pipeline was oriented horizontal such that progress through the pipeline is from top to bottom, rather than left to right. This would look better on a standard widescreen computer monitor, I think. Thoughts, @wlandau?

See example:
![image](https://user-images.githubusercontent.com/27825069/198113424-41d92c38-88f3-4da1-9b96-49459bd92be9.png)

**Reply by wlandau** [reply]
Yeah, you can try `visNetwork::visHierarchicalLayout(targets::tar_visnetwork(), direction = "UD")`. The internal code that generates the starting graph is here:

https://github.com/ropensci/targets/blob/ae3dd34173e4c92ec2d81d5c6cd38bac16e4287c/R/class_visnetwork.R#L100-L137
